### PR TITLE
Linux: Add install target to Makefile

### PR DESCRIPTION
**Pull request by Holzhaus**
Related: #795

### CODE REVIEW

File: shell/linux/Makefile [hunk 1]
```diff
@@ -179,6 +179,10 @@ INCS += -I$(RZDCY_SRC_DIR) -I$(RZDCY_SRC_DIR)/deps -I$(RZDCY_SRC_DIR)/khronos
 LIBS += -lm -lrt -ldl
 LIBS  += -lpthread
 
+ifndef PREFIX
```
Comment (skmp) [inline shell/linux/Makefile]: How about `PREFIX?=/usr/local` ?

**Comment by skmp** [comment]
I think ?= is portable enough to be used. Otherwise :+1: :D /me feels so slow at getting things done now -.-

**Comment by Holzhaus** [comment]
I also added a simple man page:
https://github.com/reicast/reicast-emulator/blob/bafd37d3f4f3b62d860b3ef823d14d5e3b508d77/shell/linux/man/reicast.1

Tell me if that's ok and I'll merge.

File: shell/linux/Makefile (hunk 1 above)
Comment (Holzhaus) [inline shell/linux/Makefile]: Fixed.

File: shell/linux/man/reicast.1 [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+Temporarily override a value of the config file.
+
+.TP
+\fB[IMAGE FILE]\fR
+Load a specific image file. If no image is given on the command line, reicast will use the value of the "image" option in the "config" section of the config file.
+
+.SH "AUTHOR"
+
+.TP
+\fBreicast\fR is developed by a number of people. The main developer is Stefanos Poiitidis (skmp).
```
Comment (skmp) [inline shell/linux/man/reicast.1]: I'd prefer more smth among the lines "\fBreicast\fR is developed by a number of people. Checkout our github page at https://github.com/reicast/reicast-emulator/graphs/contributors"

Comment (Holzhaus) [inline shell/linux/man/reicast.1]: Fixed.

**Pull request merged**